pkg/ocm/ocm.go

```diff
@@ -0,0 +1,133 @@
+package ocm
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	_ "github.com/golang/mock/mockgen/model"
+	sdkcfg "github.com/openshift-online/ocm-cli/pkg/config"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+)
+
+//go:generate mockgen -destination=./../utils/mocks/ocm_client_mock.go -package=mocks github.com/openshift/configuration-anomaly-detection/pkg/ocm OcmClient
+
+type ocmclient struct {
+	conn *sdk.Connection
+	cfg  *sdkcfg.Config
+}
+
+// NewOcmClient will create a new ocm client by using the path to a config file
+// if no path is provided, it will assume it in the default path
+func NewOcmClient(ocmConfigFile string) (ocmclient, error) {
+
+	client := ocmclient{}
+	cfg, err := newConfigFromFile(ocmConfigFile)
+	if err != nil {
+		return client, fmt.Errorf("failed to load config file: %w", err) // ConfigError{path: ocmConfigFile, err:err}
+	}
+	client.cfg = cfg
+
+	conn, err := client.establishConnection()
+	if err != nil {
+		return client, fmt.Errorf("can't create connection: %w", err) // ConnectionError{client:client, err:err}
+	}
+	client.conn = conn
+
+	return client, nil
+}
+
+// loadConfig loads the configuration file (ocmConfigFile, ~/.ocm.json, /ocm/ocm.json)
```

Comment:
The doc string does not match the function name. This breaks the go doc generation.